frontend/packages/dev-console/src/utils/application-utils.ts

```diff
@@ -305,16 +305,25 @@ export const createResourceConnection = (
     return Promise.reject();
   }
 
-  const connectValue =
-    _.get(target, ['metadata', 'labels', 'app.kubernetes.io/instance']) || target.metadata.name;
-
-  const replaceValue =
-    replacedTarget &&
-    _.get(
-      replacedTarget.metadata,
-      ['labels', 'app.kubernetes.io/instance'],
+  const connectTargetName = _.get(
+    target,
+    ['metadata', 'labels', 'app.kubernetes.io/instance'],
+    target.metadata.name,
```

Comment:
We are moving away from `lodash` and updating with optional chaining as we are updating code.